code/modules/hydroponics/seed_extractor.dm

```diff
@@ -190,11 +190,18 @@
 		if("select")
 			var/item = params["item"]
 			if(piles[item] && length(piles[item]) > 0)
-				var/datum/weakref/WO = piles[item][1]
-				var/obj/item/seeds/O = WO.resolve()
-				if(O)
-					piles[item] -= WO
-					O.forceMove(drop_location())
+				var/datum/weakref/found_seed_weakref = piles[item][1]
+				var/obj/item/seeds/found_seed = found_seed_weakref.resolve()
+				if(found_seed)
+					piles[item] -= found_seed_weakref
+					if(usr)
+						var/mob/user = usr
+						if(user.put_in_hands(found_seed))
+							to_chat(user, "<span class='notice'>[found_seed] drops into your hand.</span>")
+						else
+							to_chat(user, "<span class='notice'>[found_seed] drops onto the seed extractor.</span>")
+					else
+						found_seed.forceMove(drop_location())
+						visible_message("<span class='notice'>[found_seed] drops onto the floor.</span>", null, "<span class='hear'>You hear a soft clatter.</span>", COMBAT_MESSAGE_RANGE)
 					. = TRUE
-					//to_chat(usr, "<span class='notice'>[src] clanks to life briefly before vending [prize.equipment_name]!</span>")
```

Comment:
It looks like someone intended to add a message to the seed extractor at some point, pasted it, then commented it out and left it there.